src/components/Atoms/Config/InputBox.tsx

```diff
@@ -253,10 +249,10 @@ export default function InputBox({
           placeholder={placeholder}
           onChange={onChange}
           maxLength={maxLength}
-          className={`input-base w-full ${
+          className={`input-base w-full ${icons.length ? 'pr-16' : 'pr-4'} ${
```

Comment:
Try to set the right padding dynamically based on the length of the icon list?
Since it might be 1 or 3
Here's a quick example
![image](https://user-images.githubusercontent.com/44907675/216488845-4a799262-8933-495d-86ff-7e6a0ef6836e.png)